docs/public-networks/reference/cli/options.md

```diff
@@ -5589,15 +5589,6 @@ The synchronization mode. Use `SNAP` for [snap sync](../../concepts/node-sync.md
 
 :::
 
-:::warning Early access feature
-
-`--Xsnapsync-bft-enabled` is an early access feature available in Besu version 24.7.1 and later.
-It is not stable, so use this option with caution.
-
-Use `--Xsnapsync-bft-enabled` with `--sync-mode=SNAP` to enable snap sync in QBFT and IBFT 2.0 private networks.
-
-:::
-
```

Comment:
so if the option is being removed altogether, we want this change (deletion)

Reply:
Based on the feedback on other PR, the option is being marked as deprecated now and will be removed in future release. Will update the docs accordingly.